cmd/oras/root/pull.go

```diff
@@ -167,8 +166,12 @@ func runPull(cmd *cobra.Command, opts *pullOptions) (pullError error) {
 
 	desc, err := doPull(ctx, src, dst, copyOptions, metadataHandler, statusHandler, opts)
 	if err != nil {
+		err = oerrors.UnwrapCopyError(err)
```

Comment:
Should this be called right after using `oras.Copy` in `doPull`?

Reply:
Good catch, updated.